frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/text_input.dart

```diff
@@ -50,6 +52,7 @@ class FlowyFormTextInput extends StatelessWidget {
       onChanged: onChanged,
       onFocusCreated: onFocusCreated,
       style: textStyle ?? Theme.of(context).textTheme.bodyMedium,
+      textAlign: textAlign ?? TextAlign.center,
```

Comment:
Same as the previous, remove the `?` and assign the `TextAlign.center` as the default value.